.github/workflows/ci.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         with:
           install: '${{ matrix.projectName }}...'
           build: ${{ ( github.ref_type == 'tag' && 'false' ) || matrix.projectName }}
-          build-type: ${{ ( ( matrix.testType == 'unit:php' || matrix.testType == 'api' ) && 'backend' ) || 'full' }}
+          build-type: ${{ ( matrix.testType == 'unit:php' && 'backend' ) || 'full' }}
```

Comment:
I guess, this change is because of what happens in `plugins/woocommerce/tests/e2e-pw/global-setup.js` where UI is needed?

Reply:
Yep, that's the reason. 
I see it as temporary measure. That global setup part is not actually needed by the API tests, and I think we'll be able to also update the UI tests to not need it. But it would take too long to refactor that piece, so I'll leave it for a follow-up.

Reply:
:+1: